boards/risc-v/esp32c3/esp32c3-devkit/src/esp32c3_bringup.c

```diff
@@ -237,6 +241,15 @@ int esp32c3_bringup(void)
 
 #endif /* CONFIG_ONESHOT */
 
+#ifdef CONFIG_ESP32C3_BLE
+  ret = esp_bt_initialize();
+  if (ret)
+    {
+      syslog(LOG_ERR, "ERROR: Failed to initialize BT\n");
+      return ret;
```

Comment:
```suggestion
```
Bluetooth failure to initialize should not terminate the board bring-up procedure.